movo_common/movo_third_party/eband_local_planner/src/eband_local_planner_ros.cpp

```diff
@@ -226,21 +226,20 @@ PLUGINLIB_DECLARE_CLASS(eband_local_planner, EBandPlannerROS, eband_local_planne
 
       // instantiate local variables
       //std::vector<geometry_msgs::PoseStamped> local_plan;
-      tf::Stamped<tf::Pose> global_pose;
-      geometry_msgs::PoseStamped global_pose_msg;
+
+      geometry_msgs::PoseStamped global_pose;
       std::vector<geometry_msgs::PoseStamped> tmp_plan;
 
       // get curent robot position
       ROS_DEBUG("Reading current robot Position from costmap and appending it to elastic band.");
       if(!costmap_ros_->getRobotPose(global_pose))
-      {
-        ROS_WARN("Could not retrieve up to date robot pose from costmap for local planning.");
-        return false;
-      }
+	{
+	  ROS_WARN("Could not retrieve up to date robot pose from costmap for local planning.");
+	  return false;
+	}
 
       // convert robot pose to frame in plan and set position in band at which to append
-      tf::poseStampedTFToMsg(global_pose, global_pose_msg);
-      tmp_plan.assign(1, global_pose_msg);
+      tmp_plan.assign(1, global_pose);
```

Comment:
no need to update to msg?

Reply:
this is the same 3rd party pkg mentioned above. i just cloned the latest version of it